libs/core/src/components/pds-select/stories/pds-select.docs.stories.mdx

```diff
@@ -0,0 +1,100 @@
+import { Meta, Story, Canvas, Source, ArgsTable } from '@storybook/addon-docs';
+import { html, render } from 'lit-html';
+
+import { extractArgTypes } from '@pxtrn/storybook-addon-docs-stencil';
+
+import * as stories from './pds-select.stories.js';
+
+<Meta
+  title="components/Select"
+  component="pds-select"
+  argTypes={extractArgTypes('pds-select')}
+  args={{
+    checked: false,
+    disabled: false,
+    invalid: false,
+    required: false,
+  }}
+  subcomponents={{
+    PdsSelectOption: 'pds-select-option',
+  }}
+/>
+
+# Select
+
+Select allows users to make a single or multiple selections from a list of options.
+
+## Sub Components
+
+### Select Option
+
+A subcomponent of the select component. This component is used to define the options of the select component.
+
+Requires a unique `component-id` property to be passed.
+
+e.g. `<pds-select-option component-id="myId" value='My value'></pds-select-option>`
+<Canvas>
+<pds-select-option component-id="myId" value='My value'></pds-select-option>
+</Canvas>
+
+#### Value vs Text Content
+Requires either a `value` property or text content to be passed. If both are passed, the `value` property will be used.
+
+e.g. `<pds-select-option component-id='myId' value='my value'>My text content</pds-select-option>`
+
+<Canvas>
+<pds-select-option component-id='myId' value='my value'>My text content</pds-select-option>
+</Canvas>
+
+will render `my value` as the option value.
+
+## Guidelines
+### Accessibility
+Our select component doesn't use the native HTML `<select>`, instead ARIA is used to achieve semantic HTML behavior.
+This select is keyboard accessible following the Comboxbox and Listbox best practices from the W3C. This allows the select component to also be screen reader accessible.
+
+- Specify a unique `component-id` for  the `pds-select`. This will properly set the relationship between the menu button and listbox.
+- Specify a unique `component-id` for each `pds-select-option` to achieve form selection through ARIA roles.
+
+## Properties
+
+<ArgsTable of="pds-select" />
+
+### Default
+
+<Story story={stories.Default} />
+
+### Select
+<Canvas>
+<div style={{display: 'flex', 'flexDirection': 'column', gap: '16px'}}>
+  {/* Using only values */}
+  <pds-select component-id="combobox" label="Label">
+    <pds-select-option component-id="opt0" value="">Select an Option</pds-select-option>
```

Comment:
Maybe as a follow-up but do you think we need a way to disable an option to prevent it from being selected, but still allow it to be used as a placeholder?